assets/data/scenario/campaign/fiveChaptersDrauvCampaignAll.json

```diff
@@ -0,0 +1,195 @@
+{
+"scenarioInfo": {
+	"id": "fiveChapterDrauvCampaignAll",
+	"type": "CAMPAIGN",
+	"nameKey": "fiveChaptersDrauvAll_v3.name",
+	"blurbKey": "fiveChaptersDrauvAll_v3.blurb",
+	"newGameImagePath": "assets/ui/reskin/newCampaign_5chp.png",
+	"sortNumber": 121,
+	"rosterSlots": [
+		{
+			"archetype": "warrior",
+			"archetypeCustomizable": true,
+			"plotRoles": [ "party", "warrior" ],
+			"customHistory": {
+				"appendHistory": [
+					{
+						"id": "fiveChapterDrauvCampaign.0.rosterCustomHistory.",
+						"impliedAspects": [
+							"drauven",
+							"drauven_pc",
+							"drauvenBlood",
+							"fluentInDruvwail",
+							"cannotAge",
+							"simplePresence",
+							"ATTACHMENTS_workingRightArm",
+							"ATTACHMENTS_workingLeftArm",
+							"drauvenSkin_naturalLeftArm",
+							"drauvenSkin_naturalLeftHand",
+							"drauvenSkin_naturalRightArm",
+							"drauvenSkin_naturalRightHand",
+							"drauvenSkin_naturalLeftLeg",
+							"drauvenSkin_naturalRightLeg",
+							"drauvenSkin_naturalHead",
+							"drauvenSkin_naturalTorso",
+							"drauvenSkin_naturalTail",
+							"drauvenGear_clothes",
+							"themeSkin_crowDravenOverride",
```

Comment:
Oh! This line seems to have snuck into my commit as well - this `crowDrauvenOverride` aspect is still a work-in-progress, so we probably shouldn't have it in these yet.

Reply:
Got it. I've removed this aspect from the history of all Drauven.